assets/js/inputs.js

```diff
@@ -0,0 +1,101 @@
+// File to grab inputs the user's preferences
+// This JS only interacts with result.html and will update the inputs variables needed for the api_calls.js
+
+// Listener to add ingredients the user wants to include the the recipe
+$("#add-ingredient").on("click", function(event) {
+
+    var ingredient = $("#ingredient-to-add").val().trim();
+
+    if (ingredient == "") {
+        console.error("no ingredient specified");
+    } else {
+
+        ingredientsArr.push(ingredient);
+
+        var foodItem = $("<p>");
+
+        foodItem.attr("id", ingredient);
+        foodItem.text(ingredient);
+    
+        var foodDelete = $("<button>");
+    
+        foodDelete.attr("ingredient", ingredient);
+        foodDelete.addClass('delete');
+        foodDelete.text("x");
+        
+        foodItem = foodItem.prepend(foodDelete);
+    
+        $("#include-ingredients").append(foodItem);
+    
+        $("#ingredient-to-add").val("");
+    };
+
+});
+
+// Listener to exclude ingredients the user doesn't want in the recipe
+$("#ex-ingredient").on("click", function(event) {
+
+    var ingredient = $("#ingredient-to-exclude").val().trim();
+
+    if (ingredient == "") {
+        console.error("no ingredient specified");
+    } else {
+
+        excludeArr.push(ingredient);
+
+        var foodItem = $("<p>");
+
+        foodItem.attr("id", ingredient);
+        foodItem.text(ingredient);
+    
+        var foodDelete = $("<button>");
+    
+        foodDelete.attr("ingredient", ingredient);
+        foodDelete.addClass('delete');
+        foodDelete.text("x");
+        
```

Comment:
Not a blocker:

FYI I learned that we can add attributes and stuff like this:
                var newLink = $('<a />', {
                    id : "recipeLink",
                    name : "link",
                    href : response.hits[i].recipe.url,
                    text : response.hits[i].recipe.label
                });